javascript/packages/cli/src/types.ts

```diff
@@ -1,34 +1,70 @@
-import type { HrmpChannelsConfig, ObjectJSON } from "@zombienet/orchestrator";
+// eslint-disable-next-line
+export namespace PolkadotLaunch {
+  export interface LaunchConfig {
+    relaychain: RelayChainConfig;
+    parachains: ParachainConfig[];
+    simpleParachains: SimpleParachainConfig[];
+    hrmpChannels: HrmpChannelsConfig[];
+    types: any;
```

Comment:
Well, I have used the exact type used by the Polkadot Launch repo: https://github.com/paritytech/polkadot-launch/blob/master/src/types.d.ts to have maximum compatibility and minimum issues so I would prefer to keep it to avoid unwanted behavior in the future, wdyt ?